translations.md

```diff
@@ -270,11 +270,27 @@ Once you've got all your translations, you need to publish them. The process for
 
 These instructions were summarised from [Localization of Documentation](https://docs.readthedocs.io/en/stable/localization.html) in the readthedocs docs.
 
+## Creating a preview
+
+During the translation process, there will be points where it will be helpful to generate a preview to allow the translators and/or reviewers to see the translations in context. 
+
+### Local build and Github 
+
+1. If you haven't already, create a staging branch. 
+2. To fetch the current translations, run `tx pull -a` to fetch all, or `tx pull -l ru` to fetch a particular language. You may need to add a force flag (`tx pull -f -a` or `tx pull -f -l ru`)
```

Comment:
Rachel, I assume that the only dependency for this to all work fine is the .tx/config file being correctly set up and available on the branch? i.e. It doesn't matter if the pot files haven't been generated and are not available locally.

Reply:
I assume so, providing that the branch you fork from has the pot files.